bin/zero-parachain/node/src/chain_spec.rs

```diff
@@ -194,7 +199,7 @@ fn testnet_genesis(
 		parachain_info: parachain_subzero_runtime::ParachainInfoConfig { parachain_id: id },
 		collator_selection: parachain_subzero_runtime::CollatorSelectionConfig {
 			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
-			candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
+			candidacy_bond: cent(ZERO) * 16,
```

Comment:
Maybe not hardcoding it. It's already assigned at runtime scope - `pub ExistentialDeposits`.
- `ExistentialDeposits::get(&ZERO)`
- Import `EXISTENTIAL_DEPOSIT` variable

Reply:
Doing so will require ORML as a dependency for `node` crate. Not sure if it is a good idea.

Reply:
I tend to disagree that hardcoding is better than having an additional dependency. 
On the other side, `candidacy_bond` and `EXISTENTIAL_DEPOSIT` connection is unclear, so I think we can leave it as is.